### PR TITLE
통합 AutoNN : Status Update API 추가(epoch_summary) & TorchScript 모델 변환기 추가

### PR DESCRIPTION
**Pull request by HyunwooCho**
### PR 타입
하나 이상의 PR 타입을 선택해주세요.
- [X] 기능 추가
- [ ] 기능 삭제
- [ ] 버그 수정
- [ ] 의존성, 환경 변수, 빌드 관련 코드 업데이트
- [ ] 기타

### PR전 체크 사항
PR전 아래 사항을 확인하였는지 체크하시고 PR 생성 바랍니다.
- [X] 자신의 브랜치로 main 브랜치 최신 내용 반영후 동작 테스트 완료
- [X] 다른 작업자의 폴더내 파일 수정하지 않음.
- [X] docker-compose.yml 수정한 경우, 수정에 따른 영향 평가 

### 관련 이슈
#157
#158

### PR 반영 브랜치
ETRI-tenace-AutoNN -> main

### 변경 사항
* 매 epoch 마다 training loss/validation accuracy/elapsed time 을 요약하여 보내는 status_update API 추가
  - update_id : epoch_summary

* CodeGen과의 원활한 연동을 위한 torchscript 모델 변환기 추가

### 테스트 결과
* epoch_summary 신호 전송 확인 - PM과의 연동은 미완료
* torchscript 모델 변환 확인 - CodeGen과의 연동은 미완료

### CODE REVIEW

**Review by ML-TANGO — APPROVED** [review]
PR 내용 확인 및 승인
